src/libs/dataUseTranslation.js

```diff
@@ -1,5 +1,99 @@
 import * as fp from 'lodash/fp';
 
+const translations = {
+  hmb: {
+    code: "HMB",
+    description: 'The primary purpose of the study is to investigate a health/medical/biomedical (or biological) phenomenon or condition.',
+    manualReview: false
+  },
+  poa: {
+    code: 'POA',
+    desciption: 'The dataset will be used for the study of Population Origins/Migration patterns.',
+    manualReview: true
+  },
+  diseases: (diseases) => {
+    const diseaseArray = diseases.sort();
+    const diseaseString = diseaseArray.length > 1 ? fp.join('; ')(diseaseArray) : diseaseArray[0];
```

Comment:
I don't think you need to check for length here, pretty sure it does the right thing with arrays of any length, even 0.